components/scream/src/physics/p3/tests/p3_unit_tests.cpp

```diff
@@ -54,28 +54,45 @@ struct UnitWrap::UnitTest<D>::TestP3Func
 
   static void run()
```

Comment:
I'm creeped out that this function is just called "run". It looks like there is at least 1 other function also called "run" in this file, which seems like it should cause conflicts. Is this how we want to do things?

Reply:
These are static member functions of structs, so the full symbol is `static void TestName::run()`, making each unique. Same for `run_bfb`.

Reply:
Ah, thanks for pointing that out. I was led astray by the struct for saturation tests is named "TestP3Func", whose genericness led me to assume all functions in that file were in that single struct. I'll rename this struct unless anyone objects.

Reply:
I agree TestP3Funcs should be renamed. At the time this test was added, I don't think we had a clear (at least to me) set of standards for how to name things like this, or even for that matter how tests should be distributed across files. Now that we have clear standards, we should definitely clean things like this up.